src/NuGet/Microsoft.CodeAnalysis.CSharp.Workspaces.nuspec

```diff
@@ -27,6 +27,8 @@ Supported Platforms:
     <file src="Microsoft.CodeAnalysis.CSharp.Workspaces.xml" target="lib\portable-net45+win8" />
     <file src="Microsoft.CodeAnalysis.CSharp.Workspaces.dll" target="lib\net45" />
     <file src="Microsoft.CodeAnalysis.CSharp.Workspaces.xml" target="lib\net45" />
+    <file src="Microsoft.CodeAnalysis.CSharp.Workspaces.dll" target="lib\netstandard1.3" />
```

Comment:
Sorry I missed the fact that this depends on Microsoft.CodeAnalysis.CSharp which depends on Microsoft.CodeAnalysis.Common, so as long as all your dependencies are covered by that dependency closure it will work. However I do think you might be setting yourself up for potential trouble having Microsoft.CodeAnalysis.Common pull in all your dependencies across all your various packages, but that is your guys decision to make.